README.md

```diff
@@ -127,7 +127,7 @@ WriteString(s string) : ret int, err error
 In some applications it may make sense to define a new package that
 simply exports the file system variable for easy access from anywhere.
 
-## Using Afero's utility functions
+## Using Aferos utility functions
```

Comment:
This quote should be allowed though I am not maintainer

Reply:
Agreed